starter-code/css/main.css

```diff
@@ -4,7 +4,176 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
-body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+//*Clearfix
+.group::after {
+     content: ".";
+     display: block;
+     height: 0;
+     clear: both;
+}
+
+/* Hides from IE-mac \*/
+//* html .clearfix {height: 1%;}
+.clearfix {display: block;}
+/* End hide from IE-mac */
+
+h1, body, html, main {
   margin: 0;
 }
+
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+body {
+font-family: Helvetica, Arial, sans-serif;
+width: 100%;
+}
+
+#top-section {
+  height: 300px;
+  width: 100%;
+  background-color: #1c5380;
+  display: block;
+
+}
+#second-section {
+  display: inline-block;
+  margin-left: 500px;
```

Comment:
Always avoid using such large values for your margin and padding. This makes your entire layout dependent on a single fixed resolution. When viewing the page on another device or resolution, the values will throw your elements all over the page. There's always a way to make the page look the way you want it without resorting to extreme padding and margins.